test/app/views/editor/poll/PollEditView.spec.coffee

```diff
@@ -0,0 +1,65 @@
+PollEditView = require 'views/editor/poll/PollEditView'
+User = require 'models/User'
+Poll = require 'models/Poll'
+
+
+
+
+describe 'PollEditView ', ->
+
+
+  it 'doesnt show error when admin permissions', ->
+
+    @pollId = _.uniqueId('poll_')
+    attrs =  _.extend({}, {
+      _id: @pollId,
+      properties: {
+      name: 'poll test',
+      answers: []}
+    }, attrs)
+    @poll = new Poll(attrs)
+
+    me.clear
+    me.set({
+      _id: '1234'
+      anonymous: false
+      email: 'some@email.com'
+      name: 'Existing User',
+      permissions: ['admin']
+    })
+
+    view = new PollEditView({}, @poll.id)
+    view.render()
+    console.log(view.$el.find('#modal-error'))
```

Comment:
I'm going to separate out the two issues in different PRs and remove the test - I couldn't get the test of noty to work as I wanted.